backend/src/main/java/com/redhat/cloud/notifications/routers/models/behaviorgroup/CreateBehaviorGroupRequest.java

```diff
@@ -0,0 +1,38 @@
+package com.redhat.cloud.notifications.routers.models.behaviorgroup;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+import javax.validation.constraints.NotNull;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
It would probably be - however I'm trying to stay backwards compatible here, in a previous iteration we were accepting a BehaviorGroup instead of a `CreateBehaviorGroupRequest`

Reply:
Makes sense 👍